plants-file-storage-service/src/test/java/uk/gov/defra/plants/filestorage/service/TemplateUploadServiceTest.java

```diff
@@ -0,0 +1,492 @@
+import static uk.gov.defra.plants.filestorage.service.FileStorageServiceTestHelper.common_upload_assertions;
+import static uk.gov.defra.plants.filestorage.service.FileStorageServiceTestHelper.common_upload_setup;
+import static uk.gov.defra.plants.filestorage.service.TemplateUploadService.DESCRIPTION;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.microsoft.azure.storage.blob.BlockBlobURL;
+import com.microsoft.azure.storage.blob.DownloadResponse;
+import com.microsoft.azure.storage.blob.StorageException;
+import com.microsoft.azure.storage.blob.models.BlobDownloadHeaders;
+import com.microsoft.azure.storage.blob.models.BlockBlobUploadResponse;
+import com.microsoft.rest.v2.RestResponse;
+import io.reactivex.Flowable;
+import io.reactivex.Single;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.net.URI;
+import java.nio.ByteBuffer;
+import java.util.UUID;
+import javax.validation.ConstraintViolationException;
+import javax.ws.rs.BadRequestException;
+import javax.ws.rs.ClientErrorException;
+import javax.ws.rs.ForbiddenException;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.StreamingOutput;
+import javax.ws.rs.core.UriBuilder;
+import lombok.SneakyThrows;
+import org.glassfish.jersey.media.multipart.FormDataContentDisposition;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.defra.plants.common.eventhub.model.Event;
+import uk.gov.defra.plants.common.representation.FileType;
+import uk.gov.defra.plants.common.security.User;
+import uk.gov.defra.plants.commontest.factory.FakeVirus;
+import uk.gov.defra.plants.filestorage.FileStorageServiceConfiguration;
+import uk.gov.defra.plants.filestorage.StubAntiVirus;
+import uk.gov.defra.plants.filestorage.antivirus.AntiVirus;
+import uk.gov.defra.plants.filestorage.antivirus.AntiVirusException;
+import uk.gov.defra.plants.filestorage.enums.FileEvent;
+import uk.gov.defra.plants.filestorage.mapper.StorageExceptionMapper;
+import uk.gov.defra.plants.filestorage.provider.BlockBlobUrlProvider;
+import uk.gov.defra.plants.filestorage.provider.StreamingOutputProvider;
+import uk.gov.defra.plants.filestorage.representation.DocumentCategory;
+import uk.gov.defra.plants.filestorage.resource.TemplateResource;
+import uk.gov.defra.plants.filestorage.service.sanitise.SanitiseService;
+
+@RunWith(MockitoJUnitRunner.class)
+public class TemplateUploadServiceTest {
+
+  private static final String ORIGINAL_FILE_NAME = "original_file_name.pdf";
+  private static final String ADMIN_ID = "cd235a5b-de4b-40bf-98e7-8b4bd330ffe4";
```

Comment:
Can we please double check these ID's are only for test purpose and are safe to use in these tests?

Reply:
This is a test id only from demo data